library/aur.py

```diff
@@ -229,19 +224,27 @@ def main():
                 'default': False,
                 'type': 'bool',
             },
+            'ignore_arch': {
+                'default': False,
+                'type': 'bool',
+            },
             'aur_only': {
                 'default': False,
                 'type': 'bool',
             },
         },
+        mutually_exclusive=[['name', 'upgrade']],
         required_one_of=[['name', 'upgrade']],
+        required_by={'use': ['skip_pgp_check', 'ignore_arch']},
```

Comment:
what does this line achieve?

Reply:
This enforces that the `skip_pgp_check` and `ignore_arch` options only be used when `use` is specified, and below we make an assertion that `makepkg` is being used instead of a helper. I might be misinterpreting the intent, but I don't think you want users to be able to specify these options without explicitly saying they intend to use `makepkg`. I don't have a strong feeling about that position, though.